frontend/util.lua

```diff
@@ -1468,4 +1496,4 @@ function util.wrapMethod(target_table, target_field_name, new_func, before_callb
     return wrapped
 end
 
-return util
+return util
```

Comment:
newline please ^_^
```suggestion
return util

```

Reply:
Will do, but I see this a lot, what's the reasoning?